src/Models/HydrostaticFreeSurfaceModels/generalized_vertical_spacing.jl

```diff
@@ -116,23 +116,27 @@ update_vertical_spacing!(model, grid; kwargs...) = nothing
 @inline Δzᶠᶠᶠ(i, j, k, grid::AbstractVerticalSpacingGrid) = ℑxyᶠᶠᵃ(i, j, k, grid, Δzᶜᶜᶠ)
 @inline Δzᶠᶠᶜ(i, j, k, grid::AbstractVerticalSpacingGrid) = ℑxyᶠᶠᵃ(i, j, k, grid, Δzᶜᶜᶜ)
 
-@inline Δz_reference(i, j, k, Δz::Number) = Δz
-@inline Δz_reference(i, j, k, Δz::AbstractVector) = @inbounds Δz[k]
+#####
+##### Reference (local) vertical spacings in `r-coordinates`
+#####
+
+@inline Δr(i, j, k, Δz::Number) = Δz
+@inline Δr(i, j, k, Δz::AbstractVector) = @inbounds Δz[k]
 
-@inline Δz_reference(i, j, k, Δz::AbstractVerticalSpacing{<:Number}) = Δz.Δr
-@inline Δz_reference(i, j, k, Δz::AbstractVerticalSpacing) = @inbounds Δz.Δr[k]
+@inline Δr(i, j, k, Δz::AbstractVerticalSpacing{<:Number}) = Δz.Δr
+@inline Δr(i, j, k, Δz::AbstractVerticalSpacing) = @inbounds Δz.Δr[k]
 
-@inline Δzᶜᶜᶠ_reference(i, j, k, grid) = Δz_reference(i, j, k, grid.Δzᵃᵃᶠ)
-@inline Δzᶜᶜᶜ_reference(i, j, k, grid) = Δz_reference(i, j, k, grid.Δzᵃᵃᶜ)
+@inline Δrᶜᶜᶠ(i, j, k, grid) = Δr(i, j, k, grid.Δzᵃᵃᶠ)
+@inline Δrᶜᶜᶜ(i, j, k, grid) = Δr(i, j, k, grid.Δzᵃᵃᶜ)
```

Comment:
this is better!

I would like to know, does this mean we use the opposite notation as MOM6? I believe in MOM6 `r` is the generalized vertical coordinate and `z` is the height coordinate. It would seem with these definitions that `z` is the name of the "generalized" coordinate and `r` is the reference (which at least for z* is the height coordinate, though for other coordinates I suppose it might be something else).

Reply:
I was thinking the reference `r` is the `z*` (always fixed between 0 and -H), while the z is the moving grid defined between -H and eta. We probably need to also add an `rnode` for this grid.

Reply:
Yes for sure. It's a notation battle. `rnode` implies that "r" means the "third coordinate" (not specifically the reference coordinate).

It'd require more herculean effort for us to be able to denote "z" as "height", rather than the generalized vertical coordinate, though.

Still something to ponder. It's not impossible. I don't think we need it here.

"r" for reference makes a lot of sense. Perhaps, we can choose a different name for `rnode`. Like `ζnode` or something.